### PR TITLE
Kubernetes like issue for this repository

### PR DESCRIPTION
**Pull request by akanex115**
# Background
https://github.com/ajktown/docs/issues/52

## TODOs
- [x] Apply the similar templates for this repository.

## Checklist (Developers)
- [x] `Draft` is set for this PR
- [x] `Title` is checked
- [x] `Background` is filled
- [x] `TODOs` are filled
- [x] `Assignee` is set
- [x] `Labels` are set
- [x] `Project` is created & linked, if multiple PRs are associated to this PR
- [x] `development` is linked if related issue exists
- [x] `yarn inspect` is run
- [x] `TODOs` are handled and checked
- [x] Final Operation Check is done
- [x] Make this PR as an open PR

## Checklist (Reviewers)
- [x] Check if there are any other missing TODOs that are not yet listed
- [x] Review Code
- [x] Every item on the checklist has been addressed accordingly
- [x] If `development` is associated to this PR, you must check if every TODOs are handled

### CODE REVIEW

File: ISSUE_TEMPLATE/dependency.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      required: true
+
+  - type: textarea
+    id: todos
+    attributes:
+      label: TODOs
+      value: |
+        ```
+        - [ ] TODO: Write your TODOs here
+        ```
```
Comment (mlajkim) [inline ISSUE_TEMPLATE/dependency.yaml:23]: leave an indent for the last line of any file

File: ISSUE_TEMPLATE/enhancement.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      required: true
+
+  - type: textarea
+    id: todos
+    attributes:
+      label: TODOs
+      value: |
+        ```
+        - [ ] TODO: Write your TODOs here
+        ```
```
Comment (mlajkim) [inline ISSUE_TEMPLATE/enhancement.yaml:23]: leave an indent for the last line of any file

File: ISSUE_TEMPLATE/infrastructure.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      required: true
+
+  - type: textarea
+    id: todos
+    attributes:
+      label: TODOs
+      value: |
+        ```
+        - [ ] TODO: Write your TODOs here
+        ```
```
Comment (mlajkim) [inline ISSUE_TEMPLATE/infrastructure.yaml:23]: leave an indent for the last line of any file

File: ISSUE_TEMPLATE/performance.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      required: true
+
+  - type: textarea
+    id: todos
+    attributes:
+      label: TODOs
+      value: |
+        ```
+        - [ ] TODO: Write your TODOs here
+        ```
```
Comment (mlajkim) [inline ISSUE_TEMPLATE/performance.yaml:23]: leave an indent for the last line of any file

File: ISSUE_TEMPLATE/refactor.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+      required: true
+
+  - type: textarea
+    id: todos
+    attributes:
+      label: TODOs
+      value: |
+        ```
+        - [ ] TODO: Write your TODOs here
+        ```
```
Comment (mlajkim) [inline ISSUE_TEMPLATE/refactor.yaml:23]: leave an indent for the last line of any file

File: ISSUE_TEMPLATE/dependency.yaml (hunk 1 above)
Comment (mlajkim) [inline ISSUE_TEMPLATE/dependency.yaml:23]:
```suggestion
        ```

```

**Review by mlajkim — COMMENTED** [review]
First review

File: ISSUE_TEMPLATE/enhancement.yaml (hunk 2 above)
Comment (akanex115) [reply ISSUE_TEMPLATE/enhancement.yaml]: Handled in [b3b2ca6](https://github.com/ajktown/docs/pull/77/commits/b3b2ca6b34e456e68c86804ef1bbc7802a738963)

File: ISSUE_TEMPLATE/performance.yaml (hunk 4 above)
Comment (akanex115) [reply ISSUE_TEMPLATE/performance.yaml]: Handled in [b3b2ca6](https://github.com/ajktown/docs/pull/77/commits/b3b2ca6b34e456e68c86804ef1bbc7802a738963)

File: ISSUE_TEMPLATE/refactor.yaml (hunk 5 above)
Comment (akanex115) [reply ISSUE_TEMPLATE/refactor.yaml]: Handled in [b3b2ca6](https://github.com/ajktown/docs/pull/77/commits/b3b2ca6b34e456e68c86804ef1bbc7802a738963)

File: ISSUE_TEMPLATE/enhancement.yaml (hunk 2 above)
Comment (akanex115) [reply ISSUE_TEMPLATE/enhancement.yaml]: Handled in [56b9148](https://github.com/ajktown/docs/pull/77/commits/56b9148f14feaf5b4faef3746fd82a6c6c11ff73)

File: ISSUE_TEMPLATE/infrastructure.yaml (hunk 3 above)
Comment (akanex115) [reply ISSUE_TEMPLATE/infrastructure.yaml]: Handled in [56b9148](https://github.com/ajktown/docs/pull/77/commits/56b9148f14feaf5b4faef3746fd82a6c6c11ff73)

File: ISSUE_TEMPLATE/performance.yaml (hunk 4 above)
Comment (akanex115) [reply ISSUE_TEMPLATE/performance.yaml]: Handled in [56b9148](https://github.com/ajktown/docs/pull/77/commits/56b9148f14feaf5b4faef3746fd82a6c6c11ff73)

File: ISSUE_TEMPLATE/refactor.yaml (hunk 5 above)
Comment (akanex115) [reply ISSUE_TEMPLATE/refactor.yaml]: Handled in [56b9148](https://github.com/ajktown/docs/pull/77/commits/56b9148f14feaf5b4faef3746fd82a6c6c11ff73)

**Review by mlajkim — APPROVED** [review]
OK

**Pull request merged**